Formula/lilypond.rb

```diff
@@ -0,0 +1,61 @@
+class Lilypond < Formula
+  desc "Music notation for everyone"
+  homepage "https://lilypond.org"
+  url "https://lilypond.org/download/sources/v2.22/lilypond-2.22.1.tar.gz"
+  sha256 "72ac2d54c310c3141c0b782d4e0bef9002d5519cf46632759b1f03ef6969cc30"
+  license "GPL-3.0-or-later"
+
```

Comment:
```suggestion

  livecheck do
    url "https://lilypond.org/source.html"
    regex(/href=.*?lilypond[._-]v?(\d+(?:\.\d+)+)\.t/i)
  end

```

`brew livecheck` unable to identify versions by default, so this needs a `livecheck` block to work. The suggested `livecheck` block above checks the first-party download page for the source tarball, which links to the `stable` archive.